File: JS-Basics/03-Conditional-Statements-Lab/p04_Password_Guess.js
function passwordGess(input) {
    let password = input[0];
    if ('s3cr3t!P@ssw0rd' == password) {
        console.log('Welcome');
    } else {
        console.log('Wrong password!');
    }
}

passwordGess(["qwerty"]);
passwordGess(["s3cr3t!P@ssw0rd"]);
passwordGess(["s3cr3t!p@ss"]);

File: Programming-Basic/03-Conditional-Statements-Lab/p04_Password_Guess.java
import java.util.Scanner;

public class p04_Password_Guess {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        //      Прочитане на входни данни
//        Да се напише програма, която чете парола
//        (един ред с произволен текст), въведена от потребителя,
//        и проверява дали въведеното съвпада с фразата "s3cr3t!P@ssw0rd".
//        При съвпадение да се изведе "Welcome".
//        При несъвпадение да се изведе "Wrong password!".

        String password = scanner.nextLine();

        if (password.equals("s3cr3t!P@ssw0rd")){
            System.out.println("Welcome");
        } else {
            System.out.println("Wrong password!");
        }
//      Пресмятания

//      Печатане на резултата
    }
}
